3-platform/warehouse/spec/src/main/java/com/_6core/platform/warehouse/rest/v1/inventory/Inventory.java

```diff
@@ -0,0 +1,10 @@
+package com._6core.platform.warehouse.rest.v1.inventory;
+
+public record Inventory(
```

Comment:
Is it response? request? pls clarify. 
Stay consistent in naming of dto objects.

```suggestion
public record Inventory(
```

Reply:
It's response. Thanks.